### DISCUSSION TITLE
so-import with BPF?

### DISCUSSION THREAD

**Discussion by wdpless** [Security-Onion-Solutions/securityonion#5659] [Unsupported Versions] [answered]
I am running 2.3.70 on a distributed deployment on CentOS built using the SO ISO. 

One node is solely dedicated for PCAP import because I have systems other than SO with historical PCAP which I need the ability to quickly recall and eventually roll into the master SO solution. 

Ask: Do PCAPs run through the steno BPF when using so-import.

I have a forward node listening on the wire with the correct bpf applied but when I put the same bpf on the designated import node (technically this is a forward node not listening on the wire) the BPF filter seems to not be applied(through the SOC pcap tool I can pull up pcap which should not be stored.)

I thought about running the steno files through tcpdump to filter out unneeded pcap after index but do to the way steno stores its index and packet files this doesn't seem like a good solution.

**Comment by dougburks** [comment, marked as answer]
`so-import-pcap` doesn't actually send the pcap through steno at all. It simply places the pcap file in a directory where the pcap agent can later find it.

**Reply by wdpless** [reply]
Thank you.
